Azure/communication-ui-library-ios
# [Bugfix] Fix for a new chat thread, read receipt not sending out for messages on the first screen

## Purpose

* Fix for a new chat thread, read receipt not sending out for messages on the first screen

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test


* Create a new chat thread
* Joining this thread from 2 clients
* Use client A to send a message to client B, client A should soon see the read receipt icon besides this message.


## Other Information
